hashSet.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
Try using a boolean array of 1000*1000 it will reduce the time and space complexities of remove functions